tests/Api/Shop/TaxonsTest.php

```diff
@@ -19,33 +19,45 @@
 final class TaxonsTest extends JsonApiTestCase
 {
     /** @test */
-    public function it_gets_taxonomy_list(): void
+    public function it_gets_taxons(): void
     {
         $this->loadFixturesFromFile('taxonomy.yaml');
 
-        $this->client->request(method: 'GET', uri: '/api/v2/shop/taxons', server: self::CONTENT_TYPE_HEADER);
+        $this->client->request(
+            method: 'GET',
+            uri: '/api/v2/shop/taxons',
+            server: self::CONTENT_TYPE_HEADER,
+        );
         $response = $this->client->getResponse();
 
-        $this->assertResponse($response, 'shop/taxon/get_taxonomy_response', Response::HTTP_OK);
+        $this->assertResponse($response, 'shop/taxon/get_taxons', Response::HTTP_OK);
```

Comment:
You are not consistent within this PR in the context of using the suffix `_response`